x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -145,6 +145,10 @@ identifier
     ;
 
 identifierPattern
+    : idPattern+
+    ;
+
+idPattern
```

Comment:
Small modification to allow quoted and unquoted declarations for the name pattern.